posts/2019/11/Variable-Explorer.md

```diff
@@ -0,0 +1,73 @@
+<!--
+.. title: Variable Explorer improvements in Spyder 4
+.. slug: Variable-Explorer-improvements-in-Spyder 4
+.. date: 2019-11-12 12:00:00 UTC-05:00
+.. author: Daniel Althviz
+.. tags: Labs, Spyder
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+Spyder 4.0 (https://www.spyder-ide.org/) will be released soon and it reflects the effort of the team to improve the users' experience. It contains lots of interesting new features that we will like you to check out! In this case, we will be talking about the improvements made to the Variable Explorer.
```

Comment:
@CAM-Gerlach, please update your sugestion so that the link on the first line of this paragraph looks like this instead

    [Spyder](https://www.spyder-ide.org/) 4 will be released very soon ...

Reply:
There's a comma at the end of

> to improve the user experience**.**

Reply:
Good catch on the comma, and I'm not sure how I missed seeing the malformed link. Thanks!